src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -838,7 +835,7 @@ private async Task<string> FinalizeSubscriptionChangeAsync(ISubscriber subscribe
             {
                 try
                 {
-                    if (!isPm5864DollarThresholdEnabled && !invoiceNow)
+                    if (invoiceNow)
```

Comment:
❗ Is this change intended? This is a change in logic. On the left side, `isPm5864DollarThresholdEnabled` being `true` means that `!isPm5864DollarThresholdEnabled && !invoiceNow` always evaluates to `false`, which would make the corresponding block of code unreachable. That's no longer the case on the right hand side, however.